src/main/resources/META-INF/rewrite/assertj.yml

```diff
@@ -24,6 +24,8 @@ tags:
 recipeList:
   - org.openrewrite.java.testing.assertj.StaticImports
   - org.openrewrite.java.testing.assertj.JUnitToAssertj
+  - org.openrewrite.java.testing.assertj.SimplifyChainedAssertJAssertions
+  # TODO Reevaluate and potentially remove these three explicit recipes
   - org.openrewrite.java.testing.assertj.UseExplicitContains
   - org.openrewrite.java.testing.assertj.UseExplicitIsEmpty
   - org.openrewrite.java.testing.assertj.UseExplicitSize
```

Comment:
@AlekSimpson I'd think we can replace these three explicit recipes classes and their tests with yaml configuration and additional parameterized tests. Could you explore that to reduce how much we maintain?
```suggestion
```

Reply:
Ya I can, I think I remember looking at those recipes when starting this one. If I remember correctly they were essentially doing the same thing as this recipe but for specific methods?

Reply:
Ok I looked into those recipes and ran their tests against this new recipe and they all passed. I have commented them out in the yaml, I think its safe to remove them.

Reply:
Great! Then let's remove those classes and associated tests, and ensure those tests are covered in the parameterized tests we already have.